src/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -1,37 +1,76 @@
 import { omit } from "lodash"
-import React, { useContext, useReducer } from "react"
+import React, { useContext, useReducer, useState } from "react"
 import useDeepCompareEffect from "use-deep-compare-effect"
 import { hasFilters } from "./Utils/hasFilters"
 import { isDefaultFilter } from "./Utils/isDefaultFilter"
 import { rangeToTuple } from "./Utils/rangeToTuple"
 
+/**
+ * Initial filter state
+ */
 export const initialArtworkFilterState = {
   attribution_class: [],
-  height: "*-*",
+  // height: "*-*",
   major_periods: [],
   page: 1,
-  price_range: "*-*",
+  // price_range: "*-*",
   sort: "-decayed_merch",
-  width: "*-*",
+  // width: "*-*",
 }
 
+/**
+ * A list of all possible artwork filters across all apps
+ */
 export interface ArtworkFilters {
   acquireable?: boolean
+  artist_id?: string
   at_auction?: boolean
   color?: string
   for_sale?: boolean
-  height: string
+  height?: string
   inquireable_only?: boolean
   keyword?: string
-  major_periods: string[]
+  major_periods?: string[]
   medium?: string
   offerable?: boolean
-  page: number
+  page?: number
   partner_id?: string
-  price_range: string
-  sort: string
+  price_range?: string
+  sort?: string
   term?: string
-  width: string
+  width?: string
+}
```

Comment:
FYI If all props in an object are to be optional, you can also use the `Partial<...>` type to do so.